### PR TITLE
Fixing improperly named class

### PR DESCRIPTION
**Pull request by sineade**
:art:

Just fixing a class name I previously made incorrectly. used in this PR https://github.com/amaabca/forms2/pull/185/

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss [hunk 1]
```diff
@@ -32,11 +32,11 @@
...
     position: relative;
     padding: 0.5em 0.5em 0.5em 3em;
+  }
 
-    &--label{
-      font-size: 1em;
-      font-weight: 700;
-    }
+  &__label{
+    font-size: 1em;
```
Comment (kaytcampbell) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: Are labels not this size by default? If not, then it should be $base-font-size

**Comment by kaytcampbell** [comment]
:hankey: bump version

**Comment by sineade** [comment]
:wave:

**Comment by kaytcampbell** [comment]
:art: :art: :art:

**Pull request merged**